### ISSUE TITLE
closeMenuOnSelect=false doesn't work on mobile chrome

### ISSUE THREAD

**Issue by gafemoyano** [JedWatson/react-select#2771]
# Are you reporting a bug or runtime error?
Bug
https://codesandbox.io/s/z6xlovo34?module=/example.js

I'm not sure if this is intended or not, but the menu doesn't stay open when I use it on my mobile phone.
Android, Chrome mobile.

Thank you for this great component, let me know what I can do to help.

**Comment by jbccollins** [comment]
I suspect the fix for this will be similar to the fix for https://github.com/JedWatson/react-select/issues/2577

https://github.com/JedWatson/react-select/pull/2762/commits/69abb3f515a90354d465cdeafba13d42dc1eb094

**Comment by Tecktron** [comment]
I tested out the `2.0.0-beta.7` tag and this is still broken in chrome on touch screen devices (android & MS surface).

**Comment by Tecktron** [comment]
After some playing around, I found that setting `blurInputOnSelect={false}` fixed this issue for me. Hope that helps you as well.

**Comment by cptiwari20** [comment]
> After some playing around, I found that setting `blurInputOnSelect={false}` fixed this issue for me. Hope that helps you as well.

This helps me. I am using react-select@2.4^. I had an issue in mobile multi-select drop-down. Every time I use to select one item it closes. But in desktop it was working properly. Now after using this method. It is working fantastic.

**Comment by kulttuuri** [comment]
FYI; If anyone is looking for solution to this, this is still broken in 3.0.8. Adding `blurInputOnSelect={false}` fixed this for me. Thank you!

**Comment by skygdi** [comment]
> FYI; If anyone is looking for solution to this, this is still broken in 3.0.8. Adding `blurInputOnSelect={false}` fixed this for me. Thank you!

I used a customized way:
`constructor(props) {
    	super(props);

    	this.state = {
    		stay_open: false,
    		selected_option_sport: null,
    		recent_changed: false
    	};
	}

_onSelectSport = (value) =>{
		let stay_open = false;
		if( value!==null && value.length>0 ) stay_open = true;
		this.setState({selected_option_sport:value,stay_open:stay_open,recent_changed:true}, function () {
		});
	};

	_onSelectSportClose = () =>{
		if( this.state.selected_option_sport==null ) this.setState({stay_open:false});
		else if( this.state.recent_changed ) {
			this.setState({recent_changed:false});
		}
		else this.setState({stay_open:false});
	};

<Select
						        value={this.state.selected_option_sport}
						        onChange={this._onSelectSport}
						        isMulti={true}
						        options={options_sport}
						        placeholder="CHOOSE SPORTS"
						        isSearchable={false}
						        closeMenuOnSelect={false}
						        menuIsOpen={this.state.stay_open}
						        onMenuClose={this._onSelectSportClose}
						        onMenuOpen={this._onSelectSportOpen}
						    />
`

**Comment by bladey** [comment]
Hi all,

Thank you everyone who had a part in addressing this question.

In an effort to sustain the `react-select` project going forward, we're closing issues that appear to have been resolved via community comments.

However, if you feel this issue is still relevant and you'd like us to review it, or have any suggestions regarding this going forward - please leave a comment and we'll do our best to get back to you!

**Comment by RebootGG** [comment]
It would be nice to address this even though the workaround by @Tecktron works just fine. I've spent a bit of time searching as why it didn't work on mobile, before I came accross this issue. There are also probably some people who didn't even test it and just assumed it worked on mobile, and are still not aware of the bug. 

If this can't be implemented, we could at least update the docs with this workaround
